graph_nets/blocks.py

```diff
@@ -455,6 +456,9 @@ def _build(self, graph):
         if `graph` has `None` fields incompatible with the selected `use_edges`,
         `use_receiver_nodes`, `use_sender_nodes`, or `use_globals` options.
     """
+    if edge_model_kwargs == None:
```

Comment:
Nit: (here and everywhere) In general it is better practice to use " is None" in these cases, because that way you guarantee that it is truly a None. If you use "==" the input objects could belongs to a class that assigns a different meaning to the comparison with None.